Gemfile.lock

```diff
@@ -34,84 +29,92 @@ GEM
       i18n
       rake (>= 10.0.0)
       sshkit (~> 1.3)
-    capistrano-bundler (1.1.3)
-      capistrano (~> 3.1)
-      sshkit (~> 1.2)
-    capistrano-holepicker (0.0.1)
+    capistrano-bundle_audit (0.0.5)
+      bundler-audit
       capistrano (~> 3.0)
-      holepicker
     capistrano-one_time_key (0.0.1)
       capistrano (~> 3.0)
     capistrano-releaseboard (0.0.1)
       faraday
     coderay (1.1.0)
-    columnize (0.8.9)
     confstruct (0.2.7)
-    daemons (1.1.9)
-    debugger (1.6.8)
-      columnize (>= 0.3.1)
-      debugger-linecache (~> 1.2.0)
-      debugger-ruby_core_source (~> 1.3.5)
-    debugger-linecache (1.2.0)
-    debugger-ruby_core_source (1.3.5)
-    deprecation (0.1.0)
+    daemons (1.2.3)
+    deprecation (0.2.2)
       activesupport
     diff-lcs (1.2.5)
+    dlss-capistrano (3.2.0)
+      capistrano (~> 3.0)
+      capistrano-bundle_audit (>= 0.0.3)
+      capistrano-one_time_key
+      capistrano-releaseboard
     docopt (0.5.0)
-    dor-services (4.12.3)
-      active-fedora (~> 5.7.1)
-      activesupport (~> 3.2, >= 3.2.18)
-      confstruct (~> 0.2.2)
-      dor-workflow-service (~> 1.7, >= 1.7.1)
-      druid-tools (~> 0.3.1)
-      equivalent-xml (~> 0.2.2)
+    domain_name (0.5.25)
+      unf (>= 0.0.5, < 1.0.0)
+    dor-rights-auth (1.0.2)
+      nokogiri
+    dor-services (5.3.0)
+      active-fedora (~> 6.0)
+      activesupport (>= 3.2.18)
+      confstruct (~> 0.2.7)
+      dor-rights-auth (~> 1.0, >= 1.0.2)
+      dor-workflow-service (~> 1.8, >= 1.8.0)
+      druid-tools (~> 0.4, >= 0.4.1)
+      equivalent-xml (~> 0.5, >= 0.5.1)
       json (~> 1.8.1)
       lyber-utils (~> 0.1.2)
-      moab-versioning (= 1.3.3)
+      moab-versioning (~> 1.4.4)
+      modsulator (~> 0.0.7)
       net-sftp (~> 2.1.2)
-      net-ssh (~> 2.7.0)
+      net-ssh (~> 2.6.5)
       nokogiri (~> 1.6.0)
-      om (~> 1.8.0)
+      nokogiri-pretty (~> 0.1.0)
+      om (~> 3.0)
+      osullivan (~> 0.0.3)
       progressbar (~> 0.21.0)
-      rdf (~> 1.0.9.0)
-      rest-client (~> 1.6.7)
+      rdf (~> 1.1.7)
+      rest-client (~> 1.7)
+      retries
       rsolr-ext (~> 1.0.3)
       ruby-cache (~> 0.3.0)
-      ruby-graphviz (~> 1.0.9)
+      ruby-graphviz
       rubydora (~> 1.6.5)
-      solrizer (~> 2.0)
-      stanford-mods (~> 0.0.14)
+      solrizer (~> 3.0)
+      stanford-mods (~> 1.1)
       systemu (~> 2.6.0)
       uuidtools (~> 2.1.4)
-      validatable (~> 1.6.7)
-    dor-workflow-service (1.7.2)
-      activesupport (~> 3.2)
-      confstruct (~> 0.2.7)
+    dor-workflow-service (1.8.0)
+      activesupport (>= 3.2.1, < 5)
+      confstruct (>= 0.2.7, < 2)
+      faraday (~> 0.9.2)
+      net-http-persistent (~> 2.9.4)
       nokogiri (~> 1.6.0)
-      rest-client (~> 1.6.7)
-    druid-tools (0.3.1)
-    equivalent-xml (0.2.9)
+      rest-client (~> 1.7)
+      retries
+    druid-tools (0.4.1)
+    equivalent-xml (0.6.0)
       nokogiri (>= 1.4.3)
-    faraday (0.9.0)
+    faraday (0.9.2)
```

Comment:
gemset includes two HTTP client libs - restclient and faraday.  Not a big deal, but might be an opportunity to simplify.